Lethal-Lemurs/nodejs-bangazon-site
# Gv branch2

## Status
**READY**

## Description
There were some errors with how we set up our migration and models for the database. the relations were not set up correctly. After going to the teachers I believe we have it corrected for the most part. Definitely fixed for the products and the product type relation. I also have begun to display the product types and appropriate products in the DOM.

## Resolves Issue Number
resolves bug with migration files/models/ and relations between products and product types.

## TEST
1. drop your migrations
2. migrate
3. seed
4. check data base
5. make sure the tables are filled with data and there are no errors in your console

## Impacted Files in Application
	deleted:    migrations/20171012141344-create-user.js
	deleted:    migrations/20171023143032-create-product.js
	deleted:    migrations/20171023143923-create-pay-type.js
	deleted:    migrations/20171023144049-create-product-type.js
	deleted:    migrations/20171023144307-create-order.js
	deleted:    migrations/20171023164657-order-product.js
	modified:   models/order.js
	modified:   models/pay_type.js
	modified:   models/product.js
	modified:   models/product_type.js
	modified:   models/user.js
	modified:   routes/index.js
	deleted:    seeders/20171023145639-product-type-seed.js
	modified:   views/index.pug
        controllers/productTypeCtrl.js
	migrations/1-20171012141344-create-user.js
	migrations/2-20171023143923-create-pay-type.js
	migrations/3-20171023144049-create-product-type.js
	migrations/4-20171023143032-create-product.js
	migrations/5-20171023144307-create-order.js
	migrations/6-20171023164657-order-product.js
	routes/productTypeRoute.js
	seeders/1-20171023145639-product-type-seed.js
	views/prodtype.pug
